presidio-anonymizer/presidio_anonymizer/core/text_replace_builder.py

```diff
@@ -31,12 +31,12 @@ def get_text_in_position(self, start: int, end: int) -> str:
         return self.output_text[start:end]
 
     def replace_text_get_insertion_index(
-        self, anonymized_text: str, start: int, end: int
+            self, new_text: str, start: int, end: int
     ) -> int:
         """
-        Replace text in a specific position with the anonymized text.
+        Replace text in a specific position with the text.
 
-        :param anonymized_text:
+        :param new_text:
```

Comment:
Missing description. In particular it's important here

Reply:
Done.